### ISSUE TITLE
Build fails on iOS

### ISSUE THREAD

**Issue by albgz** [TahaMalas/internet_speed_test#3]
My iOS build is failing after I have installed this package. It fails regardless if I add a custom testserver or not.

```
Xcode build done.                                           31.1s
Failed to build iOS app
Error output from Xcode build:
↳
    ** BUILD FAILED **


Xcode's output:
↳
    /Users/macuser/development/flutter/.pub-cache/hosted/pub.dartlang.org/internet_speed_test-1.1.0/ios/Classes/SwiftInternetSpeedTestPlugin.swift:25:27: error: missing argument label 'testServer:' in call
                startListening(args: args, flutterResult: result, methodName: "startDownloadTesting", argsMap["testServer"] as! String)
                              ^
                                                                                                      testServer: 
    /Users/macuser/development/flutter/.pub-cache/hosted/pub.dartlang.org/internet_speed_test-1.1.0/ios/Classes/SwiftInternetSpeedTestPlugin.swift:28:27: error: missing argument label 'testServer:' in call
                startListening(args: args, flutterResult: result, methodName: "startUploadTesting", argsMap["testServer"] as! String)
                              ^
                                                                                                    testServer: 
    /Users/macuser/development/flutter/.pub-cache/hosted/pub.dartlang.org/internet_speed_test-1.1.0/ios/Classes/SwiftInternetSpeedTestPlugin.swift:40:13: warning: variable 'fun' was never mutated; consider changing to 'let' constant
            var fun = {
            ~~~ ^
            let
    Command CompileSwift failed with a nonzero exit code
    Command CompileSwift failed with a nonzero exit code
    note: Using new build system
    note: Building targets in parallel
```

**Comment by TahaMalas** [comment]
Thanks for opening an issue, the issue is now solved by @aks3800 due to his kind contribution

You can check the latest version of the package which is `1.1.1`.

**Comment by TahaMalas** [comment]
Another version is now released for the package `1.1.2`.
The package is consistently updated, so make sure you're always using the latest one

Cheers.
